keras/utils/data_utils.py

```diff
@@ -352,11 +353,28 @@ def __len__(self):
         raise NotImplementedError
 
 
+class HolderManager(multiprocessing.managers.BaseManager):
```

Comment:
Add class docstrings to the two new classes to explain what is going one for people reading the code. `seq` in the constructor should be documented.